apps/docs/src/components/LeftSidebar/LeftSidebar.astro

```diff
@@ -15,9 +15,9 @@ const langCode = getLanguageFromURL(currentPage);
 const sidebar = SIDEBAR[langCode];
 ---
 
-<nav class="mr-2 w-full" aria-labelledby="grid-left">
+<nav class="mr-2 mt-2 w-full" aria-labelledby="grid-left">
```

Comment:
please remove mt-2 on resolutions without mobile menu so the getting statrted is aligned with the most top conent (`introduction`)

Reply:
Okay, done